system/jlib/jfile.cpp

```diff
@@ -7999,6 +8030,10 @@ MODULE_INIT(INIT_PRIORITY_STANDARD)
         if (componentConfig->getProp("expert/@disableIFileMask", fileFlagsStr.clear()) || globalConfig->getProp("expert/@disableIFileMask", fileFlagsStr))
             expertDisableIFileFlagsMask = (IFEflags)strtoul(fileFlagsStr, NULL, 0);
 
+        safeStatBehaviour = static_cast<SafeStatBehaviour>(componentConfig->getPropInt("expert/@safeStatBehaviour", globalConfig->getPropInt("expert/@safeStatBehaviour", static_cast<unsigned>(defaultSafeStatBehaviour))));
```

Comment:
Theoretically this could cause the throwUnimplemented on line 246 - but I think that is fine.